github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1881,6 +1881,32 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
```

Comment:
It should always run, but I think we should start as optional first.
I would also prefer to see all the options explicitly set, at least those that have direct implications on the visibility and gateway (e.g. `optional`, `skip_report`).

Reply:
[Done](https://github.com/kubevirt/project-infra/compare/498d2712f9cba2c5ea9647c5cd468cd14f3b0ded..83308dc3a07b358862c995faa9e6ec6201879276)

Added `optional: true` and `skip_report:true` explicitly